sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -85,11 +90,13 @@ export class AppConfigurationClient {
     configurationSetting: ConfigurationSettingParam,
     options: ConfigurationSettingOptions = {}
   ): Promise<AddConfigurationSettingResponse> {
-    return this.client.putKeyValue(configurationSetting.key, {
-      ifNoneMatch: "*",
-      label: configurationSetting.label,
-      entity: configurationSetting,
-      ...options
+    return this.spanner.trace("addConfigurationSetting", options, (_, newOptions) => {
```

Comment:
`(_, newOptions)` looks kinda ugly to me. Do we need to return the created span for some reason? 

If so, you could either make it the second param (so you don't have to skip over it) or use an object return to destructure `{ newOptions }`

Reply:
Going with the second argument approach.

Clarify for me - is the use case with the span that I could potentially create other child operations? If so, then I'd need the created span to maintain continuity, right?

Reply:
I swapped the arguments which made things nicer and still lets code get to the span if we decide we want to create little hierarchies of calls.